### ISSUE TITLE
A11y : Cannot access edit-mode/collapsible section via keyboard

### ISSUE THREAD

**Issue by zuchka** [grafana/grafana#46488]
**Steps to reproduce**:

In edit mode > panel options tab skips whole section when collapsed (but it is accessible if expanded) 

**Actual Result**:

[see video](https://drive.google.com/file/d/17tBLmtfBGgVqRWqCN4JeS1POaYGi-sQU/view?usp=sharing)

**Expected Result**

use can expand collapsible sections via keyboard

**Relevant WCAG Criteria:**

[2.1.1 Keyboard - Level A](https://www.w3.org/WAI/WCAG21/quickref/?versions=2.0&showtechniques=335%2C332#keyboard)

**Environment**:
- Grafana version: 8.4.3
- Data source type & version:
- User OS & Browser:
- Others:
